openIt/jsonld-contexts/openIt.jsonld

```diff
@@ -4,13 +4,13 @@
     "Gate": "https://vocab.egm.io/Gate",
     "Key": "https://vocab.egm.io/Key",
     "codes": "https://vocab.egm.io/codes",
-    "for": "https://vocab.egm.io/for",
     "from": "https://vocab.egm.io/from",
+    "hasGate": "https://vocab.egm.io/hasGate",
+    "hasKey": "https://vocab.egm.io/hasKey",
```

Comment:
In that case what about we prefix with for and uses i.e: `forGate` and `usesKey`

Reply:
I think that works best.